### PR TITLE
[develop]Crypsis : Edit messages with attachments such as zip, pdf #2925

### PR DESCRIPTION
**Pull request by xillibit**

### CODE REVIEW

**Comment by 810** [comment]
if file it should be <i class="icon-flag-2"></i> not a image

**Comment by xillibit** [comment]
Can-you comment on the changes because i don't see where you have seen that ?

File: components/com_kunena/site/template/crypsis/media/js/upload.main.js [hunk 1]
```diff
@@ -214,7 +214,12 @@ jQuery(function($) {
 				if($.isEmptyObject(data.files)==false) {
 					fileCount = Object.keys(data.files).length;
 					$( data.files ).each(function( index, file ) {
-						var object = $( '<div><p><img src="'+file.path+'" width="100" height="100" /><br /><span>'+file.name+'</span><br /></p></div>' );
+						var image = '';
+						if (file.image===true) {
+							image = '<img src="'+file.path+'" width="100" height="100" /><br />';
```
Comment (810) [inline components/com_kunena/site/template/crypsis/media/js/upload.main.js]: here?

Comment (xillibit) [inline components/com_kunena/site/template/crypsis/media/js/upload.main.js]: If it's an image it display the image thumbnail else it display nothing, so what is wrong ?

Comment (810) [inline components/com_kunena/site/template/crypsis/media/js/upload.main.js]: When its no image, you must show the icon.

so also add:
````
if (file.image===false) { 
 image = '<i class="icon-flag-2"></i><br />'; 
}
````

**Comment by xillibit** [comment]
And now ?

**Comment by 810** [comment]
add also icon-big to icon class then its ok

**Comment by xillibit** [comment]
Updated

**Comment by 810** [comment]
+1

**Pull request merged**